prowler/lib/outputs/json_asff/json_asff.py

```diff
@@ -11,16 +11,17 @@
 from prowler.lib.utils.utils import hash_sha512
 
 
-def generate_json_asff_status(status: str) -> str:
+def generate_json_asff_status(status: str, muted: bool = False) -> str:
```

Comment:
- Where are you calling this function with the new muted field?
- Please add a test to check the new behaviour?